js/components/portal-dashboard/feedback/feedback-legend.tsx

```diff
@@ -5,15 +5,18 @@ import AwaitingFeedbackQuestionBadgeIcon from "../../../../img/svg-icons/awaitin
 import GivenFeedbackQuestionBadgeIcon from "../../../../img/svg-icons/given-feedback-question-badge-icon.svg";
 import UpdateFeedbackQuestionBadgeIcon from "../../../../img/svg-icons/update-feedback-question-badge-icon.svg";
 import { FeedbackLevel } from "../../../util/misc";
+import { FeedbackNoteToggle } from "./feedback-note-toggle";
+import { FeedbackSettingsToggle } from "./feedback-settings-toggle";
 
 import css from "../../../../css/portal-dashboard/feedback/feedback-legend.less";
 
 interface IProps {
   feedbackLevel: FeedbackLevel;
+  rubric?: any;
```

Comment:
Will this pick up the Rubric type if the other PR is merged?

Reply:
Yes - that PR is chained after this one.